### ISSUE TITLE
Composer issues with  1.2.0

### ISSUE THREAD

**Issue by dathwa** [rainlab/userplus-plugin#41]
Hello.
Looks like you published 1.2 today. [Change log](https://octobercms.com/plugin/rainlab-userplus#changelog)
in my composer...
```
"rainlab/user-plugin": "^2.0",
 "rainlab/userplus-plugin": "^1.1",
```
Running composer update. Not sure what to tweak. Any suggestions? Thank-you.

```
Your requirements could not be resolved to an installable set of packages.

composer update...
  Problem 1
    - Root composer.json requires rainlab/userplus-plugin ^1.1 -> satisfiable by rainlab/userplus-plugin[v1.2.0].
    - rainlab/userplus-plugin v1.2.0 requires rainlab/user-plugin ~1.0 -> found rainlab/user-plugin[v1.5.0, ..., 1.x-dev] but it conflicts with your root composer.json require (^2.0).
```

**Comment by daftspunk** [comment]
Hey @dathwa 

We pushed a fix with this change https://github.com/rainlab/userplus-plugin/commit/2a42930ec96c33166f8b12867f012585801ff049

Can you confirm that it helps?

**Comment by dathwa** [comment]
> Hey @dathwa
> 
> We pushed a fix with this change [2a42930](https://github.com/rainlab/userplus-plugin/commit/2a42930ec96c33166f8b12867f012585801ff049)
> 
> Can you confirm that it helps?

Yip. Glorious. Thank-you.
